### ISSUE TITLE
[Informational] Use case for gtrace CLI

### ISSUE THREAD

**Issue by solsu01** [moshebe/gtrace#19]
This is pretty neat! I'm the PM for Google Cloud Trace and I'm curious about your use case(s) for this CLI. When do you use this CLI as opposed to using the Google Cloud Console web UI for Cloud Trace?

I suspect the ease of being able to use cmd line tools like grep, sed, etc on a terminal is the main motivation but wanted to confirm.

**Comment by moshebe** [comment]
Hi @solsu01!
We faced some issues with the Google Cloud Trace web interface with some of our use cases, will list the major ones:
1. when you have a lot of traces within a short timeframe it's impossible to inspect them (not clickable)
2. once you pick the relevant trace there is not search option within the trace itself. also, the self is lazy-loaded so utilizing the browser search does not work also.

And, of course, we found the utilities you mentioned very useful for searching, filtering and aggregating traces from multiple projects.

Thanks for your attention, it's really appreciated!
